FND/Places/Countries.rdf

```diff
@@ -102,11 +102,65 @@
 		<skos:definition>community in which people live or have lived, without being specific as to size, population or importance</skos:definition>
 	</owl:Class>
 	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-plc-cty;hasBusinessCenter">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-plc-cty;hasMunicipality"/>
+		<rdfs:label>has business center</rdfs:label>
+		<rdfs:range rdf:resource="&fibo-fnd-plc-cty;BusinessCenter"/>
```

Comment:
It's astounding that this class is in Countries not BusinessCenters!

Reply:
It is used in BusinessDates, so it has to be in FND not FBC.

Reply:
I understand your point but most people would not. The naming and the split makes us look ridiculous.

Reply:
I've modified the abstract for the countries ontology - hopefully that will suffice.

Reply:
If not, most of the content of the business centers ontology could be moved to this countries ontology.  There is a restriction that I could move to Business dates, but that's the only exception.  Do you want me to do that?